### DISCUSSION TITLE
BTC to LND Node Transfer Not Showing yet on node's wallet after almost two weeks?

### DISCUSSION THREAD

**Discussion by BonzoDogDooDah** [lightningnetwork/lnd#8687] [Troubleshooting] [answered]
Hi, I realize that this may or may not be an question to ask of the Umbrel developers but I have to sadly report to you all that they rarely answer any posts on their forum. Please help us with this situation if you can. We sent BTC from Bitfinex to our Electrum Wallet on our Windows PC which has shown up in the Electrum's history to then be sent directly over tor 
(blue icon was active) to our Lightning Node (using Umbrel) which we have done many times now but this time it still has not arrived in our nodes BTC wallet 
yet after almost two weeks and over 2340 confirmantions as seen on mempool.  https://mempool.space/tx/97fdb66ab006da2c634f77f8dfa66d959609517a15878266c67b9686a9866894  
any ideas as to why it has yet to reach our node? BTW we sent a 2nd amount of BTC again using Bitfinex to our Electrum Wallet last Friday and that has also not shown 
up on our node's BTC wallet and yes it is recognized in the Electrum wallet but not the node?  Thank you for any assiatance anyone can offer us to try and fix this problem :)  Oh BTW our Electrum wallet has not reported any issues.

**Comment by guggero** [comment]
Sounds like your `lnd` wallet/node is out of sync. Check that it has the latest block (compared to a block explorer).

Or, just a hunch: How did you generate the address you sent to? Did you specifically ask `lnd` to generate a new one or do you have `lnd`'s xpub/seed imported somewhere and generated an address there?

If you didn't generate the address outside of `lnd`, then adding the `--reset-wallet-transactions` flag (or `reset-wallet-transactions=1` in the config file) might help. But that will take multiple hours to run.

**Reply by BonzoDogDooDah** [reply]
Hi Guggero, Thanks for the reply. We use Electrum outside of Umbrel which is connected to our node via the Electrum server app and sent BTC to a new wallet address we created in Electrum and it arrived and should then normally arrive into the node's wallet after several conformations but this time it did not arrive? Am I possibly missing a step due to old age? Thank you for your help.

**Reply by guggero** [reply]
> new wallet address we created in Electrum

No, `lnd` will not recognize funds sent to an address that it has not yet derived (e.g. with a derivation index greater than what the `lnd` internal wallet is currently tracking). So to make the funds appear, you have to re-scan with the `reset-wallet-transactions=true` option as mentioned above (can literally take days on Umbrel depending on the hardware).

To avoid this, make sure you ONLY send to addresses that you generate with `lncli newaddress` (or using a UI on Umbrel) on `lnd`.

**Reply by BonzoDogDooDah** [reply]
Thank you again for your reply. Sorry for being so dim but are you saying that the Electrum Wallet that is connected to the Umbrel Node via the Electum Server App has to send the BTC in the Electrum Wallet to an address created on the Umbrel node and that is a step I missed? Or are you saying I have to perform a re-scan with the commands mentioned above and then wait possibly a few days for the funds to arrive into the node's BTC wallet and that I have performed the correct steps? Thank you for your help.

**Reply by guggero** [reply]
> the Electrum Wallet that is connected to the Umbrel Node

I'm not sure exactly what you mean by that. I am assuming that with "connected" you just mean it gets the block/transaction information from the `bitcoind` running on your Umbrel. But I'm pretty sure every Electrum software has its own wallet, so the private keys aren't related to anything on your Umbrel node (unless you explicitly imported the `lnd` seed into Electrum, which I don't know you did, so not sure how to answer your question).

> arrive into the node's BTC wallet

What do you mean by that? The on-chain wallet or `lnd`? I know things are perhaps named a bit weird on the Umbrel, so it makes more sense to you than to me (also, I don't know how many different kinds of wallets are on Umbrel, so I'm really not sure what you mean).

> has to send the BTC in the Electrum Wallet to an address created on the Umbrel node and that is a step I missed

Yes, for future transfers to avoid this issue.

>  I have to perform a re-scan with the commands mentioned above

To resolve the current issue. Assuming that the address you sent the funds to actually belongs to `lnd` (again, I'm confused what seed you're using where and what address you sent to exactly).

**Reply by BonzoDogDooDah** [reply, marked as answer]
Hi again, Sorry for any misunderstandings I may have created. I think I have it now, thank you for your help. I believe the step I have forgotten to perform was to create a BTC receive address in the BTC onchain wallet in the Bitcoind app on our Umbrel node and then send the funds in the Electrum wallet on the PC (that is connected to the Electrum Server app on the Umbrel node) to the  BTC onchain wallet on the Umbrel node that will arrive onto the node after a time so we can then open more Lightning channels. Is that correct? Thank you so much for your help I will keep this info if I should forget the correct steps to take when funding our node in the future.

**Reply by BonzoDogDooDah** [reply]
Hi Guggero, Sorry if you have time would you please answer one more quick question? If so here it is. could we avoid using the Eletrum wallet software on our PC that connects over tor to our Umbrel node to receive and then send BTC to our Umbrel node by just sending the BTC from our exchange's BTC wallet directly to a receive wallet address on the Bitcoind app's wallet on our Umbrel node thereby removing one of the steps to funding our Umbrel node to then be able to set up new lightning channels? Oops a long quick question, sorry. Thank you once again.

**Reply by guggero** [reply]
> just sending the BTC from our exchange's BTC wallet directly

Yes of course. I don't see why you did it any differently before, unless it has something to do with the address type the exchange allows to send to. Most exchanges should support all address types (except for maybe Taproot). So just make sure you generate an address of the correct type on the Umbrel UI, then you can send directly (the exchange should tell you if it doesn't like a certain address type).

**Reply by BonzoDogDooDah** [reply]
Hi again, The Umbrel set up information mentions choosing your preferred wallet software (we choose electrum because I could not get Zap to work) and shows you how to set each one of them up so we have just followed their recommendations but since Umbrel is merely a gui for LND it should work the same as when using the cli version correct? Without the requirement of using an Electrum wallet to send funds to our node? Thank you again Guggero.
